### ISSUE TITLE
PHP missing ?

### ISSUE THREAD

**Issue by MichielioZ** [leokhoa/laragon#7]
I downloaded the 3.2 release from github...
Is it just me or is PHP missing from this release ?

**Comment by leokhoa** [comment]
@MichielioZ : You should download this package:
https://github.com/leokhoa/laragon/releases/download/3.2.3/laragon-wamp-3.2.exe

**Comment by MichielioZ** [comment]
@leokhoa : I already downloaded that package... no PHP in there for me.
Maybe it's because I extract the installer instead of a normal install ?

**Comment by MichielioZ** [comment]
I think it has to do with the extraction of the installer.
All folders and configuration laragon needs (etc, usr, www, data) of course don't get made when extracting (would have liked these to be, no need to make the installer do it).
So probably PHP needs one or more of these folders to be present ?!
Anyway: a simple portable version is all I need (no services setup etc.), any advice how to proceed ?

**Comment by leokhoa** [comment]
@MichielioZ : Laragon has a portable version. You can download it here:
https://laragon.org/download

**Comment by MichielioZ** [comment]
@leokhoa : Yes, I downloaded that package and although it has some "leftover" configuration in there and outdated PHP/MySQL, I managed to update it using the extracted wamp-3.2 installer.
It took quite a lot of "fiddling" though, so in the end I just downloaded official PHP and MariaDB packages and used AutoIt to create a simple start/stop executable. It's easier to update later on...

**Comment by leokhoa** [comment]
@MichielioZ : Haha, you don't need to do the hard ways.
 Adding new PHP versions and MySQL is a breeze with Laragon.
**PHP:**
https://forum.laragon.org/topic/166/tutorial-how-to-add-another-php-version

**MySQL/MariaDB:**
https://forum.laragon.org/topic/164/tutorial-how-to-add-another-mysql-version

You can even add new Apache, Nginx, MongoDB, Postgres the same easy way like that.
